samples/bluetooth/mesh/light_ctrl/boards/thingy53.c

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <init.h>
+
+#include <img_mgmt/img_mgmt.h>
+#include <os_mgmt/os_mgmt.h>
+#include <mgmt/mcumgr/smp_bt.h>
+
+#include <usb/usb_device.h>
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(thingy53_setup);
+
+
+static int bt_smp_init(const struct device *dev)
+{
+	ARG_UNUSED(dev);
+	int err = 0;
+
+	img_mgmt_register_group();
+	os_mgmt_register_group();
+
+	err = smp_bt_register();
+
+	if (err) {
+		LOG_ERR("SMP BT register failed (err: %d)", err);
+	}
+
+	return err;
+}
+
+static int usb_cdc_init(const struct device *dev)
```

Comment:
The function name declares initialization Communication Device Class (CDC) profile but the function does just general USB initialization. Any reasons for that?

Reply:
I did some testing and the difference is that serial output is enabled when this call is done.

Reply:
Probably these are some stranges of USB support. It is interesting what to do if I want only MSD or DFU class :)  but it is out of scope even our team activity.